publisher-sdk/proguard-rules.pro

```diff
@@ -37,43 +34,8 @@
 # Guarded by a NoClassDefFoundError try/catch and only used when on the classpath.
 -dontwarn kotlin.Unit
 
-# Top-level functions that can only be used by Kotlin.
--dontwarn retrofit2.-KotlinExtensions
-
 # Leaving Criteo SDK models.
```

Comment:
Nit: this comment is not really true now. Also, I wonder if the `keepparamnames` should be above with all the `keepattributes`.